homeWork_02/ex3_ArithmeticOperations/Artem_Nefedov_Hw2_ex3/Artem Nefedov/src/MathOperationsDemo.java

```diff
@@ -0,0 +1,47 @@
+public class MathOperationsDemo {
+    public static void main(String[] args) {
+        System.out.println("Welcome to the 'Math Operations' ");
+        int a = 82;
+        int b = 10;
+        System.out.println("1) Суммирование '+': " );
+        System.out.print("a =");
+        System.out.print(a);
+        System.out.print(", ");
+        System.out.print("b =");
+        System.out.print(b);
+        int summ = a + b;
+        System.out.println();
+        System.out.print("a + b =");
+        System.out.println(summ);
+     double c = 15.8;
+     double d = 75.5;
+        System.out.println("2) Вычитание '-': " );
+        System.out.print("с =" + c + ", d = " + d);
+        double difference = d - c;
+        System.out.println();
+        System.out.print("d - c =");
+        System.out.print(difference);
+        System.out.println();
+        System.out.println("3) Деление '/': " );
+       long e = 24;
+       long f = 4;
+        System.out.println("e =" + e + ", f =" + f);
+        long division = e / f;
```

Comment:
why long? division operation can return non-integer result and you loss a fractional part.

Reply:
code review again pls